### DISCUSSION TITLE
Can't deploy with docker compose

### DISCUSSION THREAD

**Discussion by pullya** [zitadel/zitadel#8179] [Q&A] [answered]
I am trying to run Zitadel locally using Docker Compose following the instructions at https://zitadel.com/docs/self-hosting/deploy/compose. However, the Zitadel container crashes with the error: duplicate key value violates unique constraint "unique_constraints_pkey". How can I fix this?

![image](https://github.com/zitadel/zitadel/assets/33639896/103d1d09-3925-4061-8c11-bc9db72ae38f)

**Comment by patheard** [comment, marked as answer]
Might be a partially failed install that created some of the data.  You could try starting fresh with these two commands that will stop all running Docker containers and clean up any potentially bad data:
```sh
docker rm -f $(docker ps -aq)
docker volume prune
```

Note that the above will wipe out all your Docker containers and their data, so only use it if i you don't have anything you care about in other Docker containers.

**Reply by pullya** [reply]
Thank you, the solution worked!
